src/lib.rs

```diff
@@ -65,7 +76,8 @@
 extern crate difference;
 #[macro_use] extern crate error_chain;
 
-use std::process::Command;
+use std::process::{Command, Output};
+use std::fmt;
```

Comment:
Are there any style recommendations how to order use and mod statements?
I regularly have this question - I don't know of any answer. :-)